### ISSUE TITLE
Animation lag

### ISSUE THREAD

**Issue by shitoexe** [ChiliLabs/CHIPageControl#22]
Hello. I have found one interesting thing.  While i scroll something on screen with your page control animation doesn't work and it starts only after scrolling is stopped. I have tried change mode for display link to `.commonModes` and it have worked perfectly.

```
internal func setupDisplayLink() {
        self.displayLink = CADisplayLink(target: self, selector: #selector(updateFrame))
        self.displayLink?.add(to: .current, forMode: .defaultRunLoopMode)
    }
```
Could you check it and update pod. Thank you.

**Comment by chili-ios** [comment]
Hi @shitoexe ,
Thanks for catching that. Will fix it asap.

**Comment by mihailstumkins** [comment]
@shitoexe Hi, we just released fix for this issue. It's available in latest release (0.1.2)
